### PR TITLE
Migrated test_data_self_heal_daemon_off

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated
[test_data_self_heal_daemon_off](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/arbiter/test_data_self_heal_daemon_off.py)

Updates: #292

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Comment by aujjwal-redhat** [comment]
```sh

Results for arbiter
Test nature : disruptive
TestName : test_data_self_heal_daemon_off
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     arb     |     PASS    |        00:05:09       |      NA     |
|   dist-arb  |     PASS    |        00:04:19       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   1   |
|            Tests Skipped             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 09 minutes 39 seconds
```

File: tests/functional/arbiter/test_data_self_heal_daemon_off.py [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+        self.all_mounts_procs = []
+        self.mounts = [{'client': self.client_list[0],
+                        'mountpath': self.mountpoint}]
+
+        # Setting options
+        options = {"metadata-self-heal": "off",
+                   "entry-self-heal": "off",
+                   "data-self-heal": "off"}
+        redant.set_volume_options(self.vol_name, options,
+                                  self.server_list[0])
```
Comment (nik-redhat) [inline tests/functional/arbiter/test_data_self_heal_daemon_off.py:84]: Use the `multi-option` to set all the options in one go.

File: tests/functional/arbiter/test_data_self_heal_daemon_off.py [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+        # Select bricks to bring offline
+        bricks_to_bring_offline = (redant.
+                                   select_volume_bricks_to_bring_offline(
+                                       self.vol_name, self.server_list[0]))
+
+        # Bring brick offline
+        redant.bring_bricks_offline(self.vol_name,
+                                    bricks_to_bring_offline)
```
Comment (nik-redhat) [inline tests/functional/arbiter/test_data_self_heal_daemon_off.py:113]: Check the output for this function as well

File: tests/functional/arbiter/test_data_self_heal_daemon_off.py (hunk 1 above)
Comment (aujjwal-redhat) [reply tests/functional/arbiter/test_data_self_heal_daemon_off.py:84]: Ack

File: tests/functional/arbiter/test_data_self_heal_daemon_off.py (hunk 2 above)
Comment (aujjwal-redhat) [reply tests/functional/arbiter/test_data_self_heal_daemon_off.py:113]: Ack

**Comment by aujjwal-redhat** [comment]
```sh

Results for arbiter
Test nature : disruptive
TestName : test_data_self_heal_daemon_off
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     arb     |     PASS    |        00:05:01       |      NA     |
|   dist-arb  |     PASS    |        00:04:07       |      NA     |
+-------------+-------------+-----------------------+-------------+
```

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**